kbchat/kvstore_test.go

```diff
@@ -0,0 +1,60 @@
+package kbchat
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// test doesn't make assumption about prior state
+func TestKVStore(t *testing.T) {
+	alice, dir := testBotSetup(t, "alice")
+	channel := getTeamChatChannel(t, "acme")
+	team := channel.Name
+	defer testBotTeardown(t, alice, dir)
+
+	namespace := "_test_namespace1"
+	key := "_test_key1"
+
+	// put with default revision
+	res2, err2 := alice.PutEntry(team, namespace, key, "value1", 0)
+	require.NoError(t, err2)
+	rev := res2.Revision
```

Comment:
interesting. so even if the key is there, we can roll on with this test? i guess that's fine.